.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+#### Problem Description
+*Describe the issue and explain **why** it is a problem or what the expected output should be.*
+
+#### Minimal Reproducible Example (reprex)
+*The goal of a minimal reprex is to make it as easy as possible to recreate your problem.
+If you've never heard of a reprex before, start by reading "What is a reprex",
+and follow the advice further down the page.*
+
+```R
+
+```
+
+#### Before you submit
+- [What is a reprex?](https://github.com/jennybc/reprex#what-is-a-reprex)
+- [How to create a minimal reprex.](https://github.com/tidyverse/ggplot2/blob/master/CONTRIBUTING.md#issues)
+- Please do not include session info unless it's explicitly asked for, or you've used `reprex::reprex(..., si = TRUE)` to hide it away.
+- If you have a question, ask at the [user mailing list](https://groups.google.com/forum/#!forum/ggplot2) or the [developer mailing list](https://groups.google.com/forum/#!forum/ggplot2-dev).
+- Remove this section and remove the section titles and the *hints* below the titles.
```

Comment:
If the instructions are clear, I think the section demarcations are just extra fluff.